package.json

```diff
@@ -23,7 +23,8 @@
     "postversion": "git push && git push --tags"
   },
   "dependencies": {
-    "isomorphic-fetch": "^2.2.1"
+    "isomorphic-fetch": "^2.2.1",
+    "wildlink-js-client": "^3.1.1"
```

Comment:
Can JavaScript depend on itself?

Reply:
I'm not sure why that got added to the package.json when I was updating other dependencies, but I'm going to remove it now.